### PR TITLE
DOC-1488: Edit subcommand.markdown

### PR DESCRIPTION
**Pull request by jbondpdx**
Edited the new subcommand page for Puppet Server.

### CODE REVIEW

**Comment by nwolfe** [comment]
:+1:

File: documentation/subcommands.markdown [hunk 1]
```diff
@@ -19,98 +18,95 @@ The following subcommands are provided:
...
+~~~sh
 lein <subcommand> -c /path/to/puppet-server.conf [--] [<args>]
-```
+~~~
 
-Note that if one of the `<args>` begins with a `-` (like `--version` or `-e`)
-then the intermediate `--` above is required for the argument to be applied to
-the subcommand and not leiningen. This is not necessary when running from
-packages (i.e. `puppetserver <subcommand>`).
+Note that if you are running from source, you need to separate flag arguments (such as `--version` or -`e`) with `--`, as shown above. Otherwise, those arguments will be applied to Leiningen instead of to Puppet Server. This isn't necessary when running from
```
Comment (camlow325) [inline documentation/subcommands.markdown]: Cosmetic - seems like the "-" before `e` should be inside of the accent characters - like `-e`.

**Comment by camlow325** [comment]
:+1: with just the one cosmetic comment.

File: documentation/subcommands.markdown (hunk 1 above)
Comment (jbondpdx) [inline documentation/subcommands.markdown]: Good catch, thank you! Fixed.

**Pull request merged**